docs/repo-creation-and-update-util.md

```diff
@@ -261,99 +247,46 @@ taf repo validate E:\\root\\namespace\\auth_repo
-
- ```
-E:\OLL\example\namespace
-|-- TargetRepository1
-|-- TargetRepository2
-|-- TargetRepository3
-|-- AuthenticationRepository
-```
-
-If we call the command as follows
+### `targets update_and_sign_targets`
+
+Update target files corresponding to target repositories specified through the target type parameter
+by writing the current top commit and branch name to target files corresponding to the listed repositories.
+Sign the updated files and then commit. Types are expected to be defined in `reposoitories.json`,
+as a part of custom data. This is expected to be generalized in the future since TAF should not
+expect a certain custom property to exist. If types are not specified, update all repositories specified
+in `repositories.json`.
+
+All target repositories are expected to be inside the same root directory, but do not necessarily have
+to be inside the same root directory as the authentication repository. However, if all of these
+repositories are in the same root directory, this directory does not have to be specified and is calculated
+as two directories up from the authentication repository's directory. Location of target repositories
+is then determined by appending namespace prefixed name of a repository, as listed in `repositories.json`
+to the root directory. For example, if `namespace1\repo1` is listed in `repositories.json` and
+the root directory is `E:\OLL\example`, this command will expect `E:\OLL\example\namespace1\repo1` to contain a
+target repository. The repository's current top commit and branch will then be written to the corresponding
+target file - `targets\namespace\repo1`. Once all target files are updated (for each all repositories of
+listed types, or all repositories listed in `repositories.json` if type is not provided), all corresponding target
+metadata files, as well as snapshot and timestamp are automatically signed.
+
+For example,
 
 ```bash
-taf targets update-repos-from-fs E:\\OLL\\examples\\auth_repo --add-branch
+taf targets update-and-sign-targets E:\\root\\namespace\\auth_repo --keystore E:\\keystore  --target-type html
 ```
 
-there is no need to directly set `namespace` and `library-dir` and  three target files will be created or
-updated. The resulting directory structure will be as seen below:
+will update target files corresponding to target repositories whose `custom\type` attribute in `repositories.json`
+is equal to `html`. NOTE - should be updated to be made more generic.
 
-```
-E:\OLL\example\namespace\AuthenticationRepository
-|-- targets
-     |-- namespace
-         |-- TargetsRepository1
-         |-- TargetsRepository2
-         |-- TargetsRepository3
-```
+> **_NOTE:_**  This command should be used with caution and primarily while initializing an authentication
+repository. If the target file contained additional data, that information will not be persisted. Make sure
+that the target repositories are on the correct branch before running the command.
```

Comment:
In this case, which command should we be using instead?

Reply:
Well, we should use our build process that uses TAF, but is in another repo. So we could say something like, TAF can be used to implement an automated process which will update all repositories in accordance to a specific project's needs if that makes sense

Reply:
Yes, that makes sense.